### ISSUE TITLE
tf.keras.callbacks.ProgbarLogger not works as expected

### ISSUE THREAD

**Issue by zhangyi02** [tensorflow/tensorflow#33669]
I was tried to use tf.keras instead of keras, and I found that tf.keras.callbacks.ProgbarLogger not works as expected, or as the old keras did.
In keras, at the end of one-epoch's training, the ProgbarLogger will just wait until the validation done, and the final info on the screen would be like 
>[training-progress-bar] training-metrics validation-metrics

But In keras, at the end of one-epoch's training, the training-progress-bar will disappear and the validation-progress-bar will show on the screen. And when validation progress finished, the final info on the screen would be like 
> [validation-progress-bar] validation-metrics 
> [training-progress-bar] training-metrics validation-metrics

 As I expected, the training-progress-bar should stay on the screen during validation, and the final info should be 
> [training-progress-bar] training-metrics
> [validation-progress-bar] validation-metrics

**Comment by gadagashwini** [comment]
@zhangyi02, Please provide the standalone code to reproduce the reported issue. Thanks!

**Comment by gadagashwini** [comment]
@zhangyi02, Any update code snippet.

**Comment by zhangyi02** [comment]
```python
import numpy as np
import keras
print(keras)


class FakedataSequence(keras.utils.Sequence):
    def __init__(self, batch_size, num_batches, stage='tr'):
        self.batch_size = batch_size
        self.num_batches = num_batches
        self.stage = stage

    def __len__(self):
        return self.num_batches

    def __getitem__(self, item):
        if self.stage == 'tr':
            np.random.seed(item)
        else:
            np.random.seed(100000 + item)
        data = np.random.random((self.batch_size, 10))
        label = np.random.randint(0, 2, size=self.batch_size)
        return data, label


def toy_model():
    x = keras.layers.Input(shape=(10,))
    y = keras.layers.Dense(1, activation='sigmoid')(x)
    return keras.models.Model(x, y)


if __name__ == '__main__':
    tr_sequence = FakedataSequence(32, 100, 'tr')
    cv_sequence = FakedataSequence(32, 10, 'cv')
    model = toy_model()
    model.compile(optimizer='sgd', loss='mse')
    history = model.fit_generator(generator=tr_sequence,
                                  epochs=10,
                                  verbose=1,
                                  validation_data=cv_sequence,
                                  workers=32)
```
while using the old keras, as `print(keras)` returns `<module 'keras' from '~/.local/lib/python3.6/site-packages/keras/__init__.py'>`, the logs looks like:
>Epoch 1/10
100/100 [==============================] - 0s 3ms/step - loss: 0.2649 - val_loss: 0.2828
Epoch 2/10
100/100 [==============================] - 0s 3ms/step - loss: 0.2648 - val_loss: 0.2820
Epoch 3/10
100/100 [==============================] - 0s 2ms/step - loss: 0.2641 - val_loss: 0.2809
Epoch 4/10
100/100 [==============================] - 0s 4ms/step - loss: 0.2659 - val_loss: 0.2802
Epoch 5/10
100/100 [==============================] - 0s 3ms/step - loss: 0.2638 - val_loss: 0.2797
Epoch 6/10
100/100 [==============================] - 0s 2ms/step - loss: 0.2648 - val_loss: 0.2790
Epoch 7/10
100/100 [==============================] - 0s 2ms/step - loss: 0.2627 - val_loss: 0.2787
Epoch 8/10
100/100 [==============================] - 0s 2ms/step - loss: 0.2634 - val_loss: 0.2781
Epoch 9/10
100/100 [==============================] - 0s 1ms/step - loss: 0.2634 - val_loss: 0.2778
Epoch 10/10
100/100 [==============================] - 0s 1ms/step - loss: 0.2632 - val_loss: 0.2776

And if change the `import keras` to `from tensorflow import keras`, and rerun this script, while `print(keras)` returns `<module 'tensorflow._api.v1.keras' from '~/.local/lib/python3.6/site-packages/tensorflow/_api/v1/keras/__init__.py'>`, the logs looks like:
>10/10 [==============================] - 0s 3ms/step - loss: 0.2566
100/100 [==============================] - 1s 11ms/step - loss: 0.2571 - val_loss: 0.2566
Epoch 2/10
10/10 [==============================] - 0s 1ms/step - loss: 0.2566
100/100 [==============================] - 0s 3ms/step - loss: 0.2568 - val_loss: 0.2566
Epoch 3/10
10/10 [==============================] - 0s 2ms/step - loss: 0.2640
100/100 [==============================] - 0s 4ms/step - loss: 0.2562 - val_loss: 0.2640
Epoch 4/10
10/10 [==============================] - 0s 2ms/step - loss: 0.2491
100/100 [==============================] - 0s 3ms/step - loss: 0.2554 - val_loss: 0.2491
Epoch 5/10
10/10 [==============================] - 0s 1ms/step - loss: 0.2500
100/100 [==============================] - 0s 4ms/step - loss: 0.2542 - val_loss: 0.2500
Epoch 6/10
10/10 [==============================] - 0s 1ms/step - loss: 0.2598
100/100 [==============================] - 0s 5ms/step - loss: 0.2545 - val_loss: 0.2598
Epoch 7/10
10/10 [==============================] - 0s 688us/step - loss: 0.2496
100/100 [==============================] - 0s 3ms/step - loss: 0.2557 - val_loss: 0.2496
Epoch 8/10
10/10 [==============================] - 0s 2ms/step - loss: 0.2510
100/100 [==============================] - 0s 4ms/step - loss: 0.2544 - val_loss: 0.2510
Epoch 9/10
10/10 [==============================] - 0s 1ms/step - loss: 0.2514
100/100 [==============================] - 0s 4ms/step - loss: 0.2549 - val_loss: 0.2514
Epoch 10/10
10/10 [==============================] - 0s 2ms/step - loss: 0.2520
100/100 [==============================] - 0s 4ms/step - loss: 0.2550 - val_loss: 0.2520

Well, I notice that even "Epoch 1/10" also disappeared.
I supposed the logs should be like:
>Epoch 1/10
100/100 [==============================] - 1s 11ms/step - loss: 0.2571
10/10 [==============================] - 0s 3ms/step - val_loss: 0.2566
Epoch 2/10
....

**Comment by gadagashwini** [comment]
Issue is replicating with Tensorflow 1.15.
Please see the gist [here](https://colab.sandbox.google.com/gist/gadagashwini/9b7726a8a9845100d2f8b18a9da47ed4/untitled252.ipynb). Thanks!

**Comment by zhangyi02** [comment]
@gadagashwini I tried TF2.0，it works just like the origin Keras. By the way, I'm using TF1.13.1. Maybe it's time change to TF2.0……

**Comment by jvishnuvardhan** [comment]
@zhangyi02 I agree that the output from `TF1.x` doesn't look as good as the output from `TF2.0`. As `TF1.15` is the last version in `TF1.x` series, I don't think there is any more updates to that branch unless there is any security related issues.

If possible, I would suggest you to start using `TF2.0`. Please close the issue, if you don't have any further questions. Thanks!
